src/Middleware/RequestQueue/src/Microsoft.AspNetCore.RequestQueue.csproj

```
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>ASP.NET Core middleware for caching HTTP responses on the server.</Description>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <IsAspNetCoreApp>true</IsAspNetCoreApp>
+    <NoWarn>$(NoWarn);CS1591</NoWarn>
```

```suggestion
    <WarningsNotAsErrors>$(WarningsNotAsErrors);CS1591</WarningsNotAsErrors>
```

Instead of suppressing undocumented API, I recommend just making this an error instead. This will helps you find undocumented API without failing the build.

---

Yeah, let's start things off right and document the public APIs as we go :)